Add stylelint-disable-line command; closes #763

davidtheclark opened the pull request.

jeddy3 commented on docs/user-guide/configuration.md:
````diff
@@ -203,6 +203,14 @@ Or you can turn off individual rules:
+
+```css
+#id { /* stylelint-disable-line */
+  color: pink !important; /* declaration-no-important */
````
Should this be `stylelint-disable-line declaration-no-important`

jeddy3 commented:
Nice. Thanks for doing this one!

I was also thinking a unique command (`stylelint-disable-line`) would probably be the best way to go about this.

In the [original issue](https://github.com/stylelint/stylelint/issues/763) there is a request for supporting `// ` disable comments. I think it's perfectly valid that standard css comments are required for these stylelint commands, and I think, as you said, less little need to add more to the codebase for a little sugar. How do you feel about that?

davidtheclark commented:
I fixed the mistake in the docs. Good catch.

`//` comments should work just fine because postcss-scss parses them as comments. I added a test to demonstrate.

(`//` comments should only raise challenges when we target comments by some means other than PostCSS's parsed AST, like string searches that hunt for `/*`. Because when we are actually looking for comment nodes, those could have originated in regular or `//` comments -- either way.)

jeddy3 commented:
> (// comments should only raise challenges when we target comments by some means other than PostCSS's parsed AST, like string searches that hunt for /*.

Understood. I thought as much, but I wasn't entirely sure. Thanks for adding the test in to demonstrate :)

This looks great to me now. Thanks again.
